sys/openbsd/sys.txt.const

```diff
@@ -65,82 +65,6 @@ RUSAGE_SELF = amd64:0
 SF_APPEND = amd64:262144
 SF_ARCHIVED = amd64:65536
 SF_IMMUTABLE = amd64:131072
-SYS_acct = amd64:51
```

Comment:
You just regenerated files and all these consts went away?
This means these .const files were havily altered manually?

Reply:
It seems WAI -- see https://github.com/google/syzkaller/pull/3836
These consts should have been removed in that PR